### PR TITLE
Update ultralytics tutorial + notebook

### PR DESCRIPTION
**Pull request by liopeer**
## What has changed and why?
- update YOLO tutorials with dedicated dataset

## How has it been tested?
- notebook ran locally

## Did you update [CHANGELOG.md](../CHANGELOG.md)?

- [ ] Yes
- [x] Not needed (internal change)

## Did you update the documentation?

- [ ] Yes
- [ ] Not needed (internal change without effects for user)
- [x] It is purely documentation.

### CODE REVIEW

File: docs/source/tutorials/yolo/index.md [hunk 1]
```diff
@@ -20,7 +20,72 @@ Install the required packages:
...
+```
+````
+
+And just like that you pretrained a YOLO11s backbone! 🥳 This backbone can't solve any task yet, so in the next step we will finetune it on the PASCAL VOC dataset.
+
+## Finetuning on PASCAL VOC
+
+### Download the PASCAL VOC Dataset
+
+And just like that you pretrained a YOLO11s backbone! 🥳 This backbone can't solve any task yet, so in the next step we will finetune it for object detection on the PASCAL VOC dataset.
```
Comment (yutong-xiang-97) [inline docs/source/tutorials/yolo/index.md]: Duplicated paragraph

File: docs/source/tutorials/yolo/index.md [hunk 2]
```diff
@@ -20,7 +20,72 @@ Install the required packages:
...
+
+## Finetuning on PASCAL VOC
+
+Now that the pretrained model has been exported, we will further fine-tune the model on the task of object detection. The exported model already has exactly the format that Ultralytics' YOLO expects, so after getting the dataset ready, we can get started with only a few lines! ⚡️ In addition to fine-tuning the pretrained model we will also train a model that we initialize with random weights. This will let us compare the performance between the two, and show the great benefits of pretraining.
```
Comment (yutong-xiang-97) [inline docs/source/tutorials/yolo/index.md]: Should I expect a new paragraph after the ⚡️ emoji?

File: docs/source/tutorials/yolo/results_VOC.png
Comment (yutong-xiang-97) [inline docs/source/tutorials/yolo/results_VOC.png:1]: For the legend I would prefer mentioning "pretrain" instead of "finetune"

File: docs/source/tutorials/yolo/index.md [hunk 3]
```diff
@@ -112,61 +173,76 @@ fig.show()
...
 ````{tab} Command Line
 ```bash
-lightly-train train out="out/my_experiment" data="<DATASET-DIR>/VOC/images/train2012" model="ultralytics/yolo11s.yaml" epochs=100 batch_size=64
+yolo detect train model="yolo11s.yaml" data="VOC.yaml" epochs=30 project="logs/voc_yolo11s" name="from_scratch"
 ```
+````
 
-```bash
-yolo detect train model="out/my_experiment/exported_models/exported_last.pt" data="VOC.yaml" epochs=100
+We can gather the validation results of both models from the logs:
```
Comment (yutong-xiang-97) [inline docs/source/tutorials/yolo/index.md]: Missing section title "Evaluating the Model Performance" as in the notebook.

File: docs/source/tutorials/yolo/index.md [hunk 4]
```diff
@@ -112,61 +173,76 @@ fig.show()
...
-
-We will use LightlyTrain to pretrain a YOLO11 model.
-
-The following scripts or CLI commands will:
-
-- Initialize a YOLO11s model with random weights.
-- Pretrain the YOLO11s model on the training images of PASCAL VOC using distillation pretraining.
-- Export the pretrained YOLO11s model.
-- Fine-tune the pretrained model on PASCAL VOC dataset using labels.
+### Finetuning the Pretrained Model
```
Comment (yutong-xiang-97) [inline docs/source/tutorials/yolo/index.md:170]: Missing the sentence below the section title as in the notebook.

File: docs/source/tutorials/yolo/index.md [hunk 5]
```diff
@@ -112,61 +173,76 @@ fig.show()
...
 ```
+````
+
+````{tab} Command Line
+```bash
+yolo detect train model="out/my_experiment/exported_models/exported_last.pt" data="VOC.yaml" epochs=30 project="logs/voc_yolo11s" name="from_pretrained"
+```
+````
+
+### Finetuning the Randomly Initialized Model
```
Comment (yutong-xiang-97) [inline docs/source/tutorials/yolo/index.md:195]: Missing the sentence below the section title as in the notebook.

File: docs/source/tutorials/yolo/index.md [hunk 6]
```diff
@@ -20,7 +20,72 @@ Install the required packages:
...
+
+### Download the PASCAL VOC Dataset
+
+And just like that you pretrained a YOLO11s backbone! 🥳 This backbone can't solve any task yet, so in the next step we will finetune it for object detection on the PASCAL VOC dataset.
+
+## Finetuning on PASCAL VOC
+
+Now that the pretrained model has been exported, we will further fine-tune the model on the task of object detection. The exported model already has exactly the format that Ultralytics' YOLO expects, so after getting the dataset ready, we can get started with only a few lines! ⚡️ In addition to fine-tuning the pretrained model we will also train a model that we initialize with random weights. This will let us compare the performance between the two, and show the great benefits of pretraining.
+
+Expect again a run-time of around 1h each for fine-tuning from the pretrained model as well as fine-tuning from randomly initialized weights.
```
Comment (yutong-xiang-97) [inline docs/source/tutorials/yolo/index.md]: What hardware config did you base your 1h estimation to? On my MacBook pro it takes ~3hrs.

File: examples/notebooks/ultralytics_yolo.ipynb [hunk 7]
```diff
@@ -163,114 +251,91 @@
...
    "outputs": [],
    "source": [
     "import matplotlib.pyplot as plt\n",
     "import pandas as pd\n",
     "\n",
-    "res_scratch = pd.read_csv(\"logs/voc_yolo11s/from_scratch/results.csv\")\n",
-    "res_finetune = pd.read_csv(\"logs/voc_yolo11s/from_pretrained/results.csv\")\n",
+    "# res_scratch = pd.read_csv(\"logs/voc_yolo11s/from_scratch/results.csv\")\n",
+    "res_scratch = pd.read_csv(\n",
+    "    \"/home/lionel/Documents/lightly-train/docs/source/tutorials/yolo/logs/voc_yolo11s/from_scratch/results.csv\"\n",
```
Comment (yutong-xiang-97) [inline examples/notebooks/ultralytics_yolo.ipynb]: Here you need to anonymize your path

File: examples/notebooks/ultralytics_yolo.ipynb [hunk 8]
```diff
@@ -163,114 +251,91 @@
...
+    ")\n",
+    "# res_finetune = pd.read_csv(\"logs/voc_yolo11s/from_pretrained/results.csv\")\n",
+    "res_finetune = pd.read_csv(\n",
+    "    \"/home/lionel/Documents/lightly-train/docs/source/tutorials/yolo/logs/voc_yolo11s/from_pretrained/results.csv\"\n",
```
Comment (yutong-xiang-97) [inline examples/notebooks/ultralytics_yolo.ipynb]: Same as above.

**Review by yutong-xiang-97 — CHANGES_REQUESTED** [review]
All good but some lint should be dealt with.

Also in line 106 in the doc and the corresponding section in the notebook, we have `Inside that directory ()` displayed in the docs without the `<DATASET-PATH>` in the brackets

File: docs/source/tutorials/yolo/index.md (hunk 2 above)
Comment (liopeer) [reply docs/source/tutorials/yolo/index.md]: Thx!

File: docs/source/tutorials/yolo/index.md (hunk 6 above)
Comment (liopeer) [reply docs/source/tutorials/yolo/index.md]: That came from the notebook. I'll remove it from here.

File: docs/source/tutorials/yolo/index.md (hunk 1 above)
Comment (liopeer) [reply docs/source/tutorials/yolo/index.md]: removed!

File: docs/source/tutorials/yolo/results_VOC.png
Comment (liopeer) [reply docs/source/tutorials/yolo/results_VOC.png:1]: fixed

File: docs/source/tutorials/yolo/index.md (hunk 3 above)
Comment (liopeer) [reply docs/source/tutorials/yolo/index.md]: added!

File: docs/source/tutorials/yolo/index.md (hunk 4 above)
Comment (liopeer) [reply docs/source/tutorials/yolo/index.md:170]: Added!

File: docs/source/tutorials/yolo/index.md (hunk 5 above)
Comment (liopeer) [reply docs/source/tutorials/yolo/index.md:195]: Added!

File: examples/notebooks/ultralytics_yolo.ipynb (hunk 7 above)
Comment (liopeer) [reply examples/notebooks/ultralytics_yolo.ipynb]: Thx, missed that!
